pkg/storaged/crypto-keyslots.jsx

```diff
@@ -558,11 +548,8 @@ function edit_passphrase_dialog(block, key) {
         Fields: [
             PassInput("old_passphrase", _("Old passphrase"),
                       { validate: val => !val.length && _("Passphrase cannot be empty") }),
-            Skip("medskip"),
-            PassInput("new_passphrase", _("New passphrase"),
-                      { validate: val => !val.length && _("Passphrase cannot be empty") }),
-            PassInput("new_passphrase2", _("Repeat passphrase"),
-                      { validate: (val, vals) => vals.new_passphrase.length && vals.new_passphrase != val && _("Passphrases do not match") })
+            NewPassInput("new_passphrase", _("New passphrase"),
+                         { validate: val => !val.length && _("Passphrase cannot be empty") }),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230608-155257-544e3ca1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)